omxplayer/player.py

```diff
@@ -428,6 +429,14 @@ def quit(self):
             logger.debug('SIGTERM Sent to pid: %s' % self._process.pid)
         except OSError:
             logger.error('Could not find the process to kill')
+ 
+    @_check_player_is_active
+    def get_filename(self):
+        """
+        Returns:
+            str: filename currently playing
+        """
+	return str(self._filename)
```

Comment:
Since we're passing in the filename there is no need to call `str` on it, filenames are inherently strings.